java/amazon-kinesis-producer-sample/pom.xml

```diff
@@ -56,7 +56,7 @@
         <dependency>
             <groupId>com.amazonaws</groupId>
             <artifactId>amazon-kinesis-producer</artifactId>
-            <version>0.14.8</version>
+            <version>0.14.9</version>
```

Comment:
Wasn't there another place we need to make this change except the pom files ?

Reply:
We need to update the cpp core, sop was outdated... added in next revision.